spec/features/chefs/show_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe 'Chef Show Page' do
+  before do
+    @chef1 = Chef.create(name: 'Gordon Ramsay')
+    @dish1 = @chef1.dishes.create(name: 'Pizza', description: 'A delicious pizza')
+    @pepperoni = Ingredient.create(name: 'Pepperoni', calories: 100)
+    @cheese = Ingredient.create(name: 'Cheese', calories: 50)  
+    @dough = Ingredient.create(name: 'Dough', calories: 200)
+    @sauce = Ingredient.create(name: 'Sauce', calories: 75)
+    @chicken = Ingredient.create(name: 'Chicken', calories: 150)
+    DishIngredient.create(dish_id: @dish1.id, ingredient_id: @pepperoni.id)
+    DishIngredient.create(dish_id: @dish1.id, ingredient_id: @cheese.id)
+    DishIngredient.create(dish_id: @dish1.id, ingredient_id: @dough.id)
+    DishIngredient.create(dish_id: @dish1.id, ingredient_id: @sauce.id)
+    visit chef_path(@chef1)
+  end
+
+  describe 'as a visitor' do
+    it 'has a link to all a chefs ingredients' do
+      expect(page).to have_content("Ingredients")
+
+      click_link("Ingredients")
+
+      expect(current_path).to eq(chef_ingredients_path(@chef1))
+    end
```

Comment:
I would also like to see testing for the content on this page though. What ingredients should we see? In thinking about this test, we want to see no duplicates, so it would be good in our test setup to ensure that at least one ingredient is being used 2+ times and then to check that it only appears once on the page.